src/components/Register/Register.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import PopupWithForm from '../PopupWithForm/PopupWithForm';
+import FormField from '../FormField/FormField';
+import { useFormWithValid } from '../../hooks/useFormWithValid';
+
+function Register(props) {
+  const { values, handleChange, errors, isValid, resetForm } =
+    useFormWithValid();
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    props.onSubmit({ values, resetForm });
+  }
+
+  return (
+    <div>
```

Comment:
empty `<div>` is not needed here. It makes no sense

Reply:
done